cirq/google/common_serializers.py

```diff
@@ -233,7 +257,7 @@ def _near_mod_2(e, t, atol=1e-8):
                                          gate_getter='phase_exponent'),
         ],
         can_serialize_predicate=lambda x: _near_mod_2(
-            cast(ops.PhasedXPowGate, x).exponent, 1)),
+            cast(ops.PhasedXPowGate, x.gate).exponent, 1)),
```

Comment:
Maybe we can change the arg name to `op` in these lambdas (same thing applies throughout this PR). Could also do this in a future change.

Reply:
Done.